Software/a314fs/a314fs.py

```diff
@@ -144,6 +145,7 @@ def send_reset(stream_id):
 ACTION_WRITE_PROTECT    = 1023
 ACTION_EXAMINE_FH       = 1034
 ACTION_UNSUPPORTED      = 65535
+ACTION_DISK_SIZE_INFO   = 1000 
```

Comment:
Could you pick some other number for this constant, that is not likely to collide with existing constants? Like perhaps 4000? And put it in the list in order (above ACTION_UNSUPPORTED). Also update the constant in a314fs.c.

Reply:
I guess 1000 specifically is not used, but many other numbers in that range (around 1000, for example 1001, 1002) are used, so it seems unnecessary to pick a number that close, when all numbers between 0..65535 are possible.